cdap-ui/app/services/app-level-loading-service.js

```diff
@@ -38,4 +38,14 @@ angular.module(PKG.name + '.services')
       }
     };
 
+    this.hideLoadingIconImmediate = function(){
+      if (!deferred) {
+        return $q.when(true);
+      } else {
+        EventPipe.emit('hideLoadingIcon.immediate');
+        deferred.resolve(true);
+        deferred = null;
+      }      
+    }
```

Comment:
Missing `;` . We should always run `gulp watch` or `gulp lint` before creating a PR. This could potentially break the UI build.